x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/save_with_errors_confirmation/index.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
```

Comment:
@e40pud 

is this fine user that user sees some error, but can't go to Definition tab or see error itself?

<img width="2375" alt="Screenshot 2024-08-29 at 10 24 40" src="https://github.com/user-attachments/assets/d0a03e4c-82e2-473d-99c0-a80f3d196a48">

I imagine it will be a case until full support of prebuilt rule editing will be released.
But before that, it can affect Serverless releases

Reply:
What you see right now is the current behaviour on both ESS and Serverless. Users are unable to modify prebuilt rules. The whole ["rule customization"](https://github.com/elastic/security-team/issues/8056) effort will allow user to do that.

Reply:
I am aware, prebuilt rule can't be modified. 
It's the fact that we show user step is invalid without showing actual error and additional information how that can affect rule's behavior.

Is there a way for user in this situation to see actual error?

Reply:
As discussed here is the ticket to discuss this behaviour https://github.com/elastic/kibana/issues/191832